cove_360/lib/threesixtygiving.py

```diff
@@ -197,7 +197,7 @@ def process(self, grant, grant_flat, path_prefix):
             pass
 
     def produce_message(self):
-        return "One or more of your grants have a Recipient Org:Identifier that is has a prefix of '360G'. If the grant is from a recipient organisation that has an external identifier (such as a charity number, company number, or in the case of local authorities, geocodes), then this should be used instead. If no other identifier can be used, then this notice can be ignored."
+        return "One or more of your grants have a Recipient Org:Identifier with a '360G' prefix. If the grant is from a recipient organisation that has an external identifier (such as a charity number, company number, or in the case of local authorities, geocodes), then this should be used instead. If no other identifier can be used, then this notice can be ignored."
```

Comment:
I think "....have a Recipient Org:Identifier that starts '360G-'" would be even clearer here. The language of 'prefix' probably isn't necessary here.